.circleci/config.yml

```diff
@@ -74,17 +74,14 @@ jobs:
       - setup_build_environment
       - gradle:
           args: :tag
-      - run:
-          name: Remove trigger tag
-          command: git tag -d release-$(git describe --abbrev=0)
       - gradle:
           args: dockerPushImages
```

Comment:
This line will fail with multiple tags present

Reply:
added step to delete the tag from local workspace.